specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -163,40 +163,33 @@
       "description": "The created subscription object.",
       "properties": {
         "subscriptionLink": {
-          "description": "The link to the new subscription.",
+          "description": "The link to the new subscription. Use this link to check the status of subscription creation operation.",
           "type": "string"
         }
       }
     },
     "SubscriptionCreationParameters": {
-      "description": "Subscription Creation Parameters required to create a new Azure subscription.",
+      "description": "The parameters required to create a new subscription.",
       "properties": {
         "displayName": {
-          "description": "The display name of the subscription.",
+          "description": "Required. The friendly name of the subscription.",
           "type": "string"
         },
         "billingProfileId": {
-          "description": "The ARM id of the billing profile.",
+          "description": "Required. The ARM ID of the billing profile for which you want to create the subscription.",
           "type": "string"
         },
         "skuId": {
-          "description": "The commerce id of the sku.",
+          "description": "Required. The SKU ID of the Azure plan. Azure plan determines the pricing and service-level agreement of the subscription.  Use 001 for Microsoft Azure Plan and 002 for Microsoft Azure Plan for DevTest.",
           "type": "string"
         },
         "costCenter": {
-          "description": "optional customer cost center",
+          "description": "Optional. If set, the cost center will show up on the Azure usage and charges file.",
           "type": "string"
         },
         "owner": {
-          "description": "rbac owner of the subscription",
+          "description": "Optional. If specified, the AD principal will get owner access to the subscription, along with the user who is performing the create subscription operation",
           "$ref": "#/definitions/AdPrincipal"
-        },
-        "additionalParameters": {
```

Comment:
Is there a reason why we are removing this parameter?

Reply:
We are not using this parameter anywhere in the API. I think it got added in the swagger in first place because in actual code, same request class is being used by  multiple APIs.

Reply:
The generated `SubscriptionCreationParameters` model in the SDKs would have included additional properties (a dictionary in C# SDK, for example) as a result of this. This change will cause the generated SDKs to no longer have that extra property in the generated models. Would any of our customers have a reason to set that? Will they be broken by the change?

Also, how does your backend handle the extra properties?

Reply:
Added back the additional parameter to be on safe side but we are not using this parameter anywhere on backend